x/interchainstaking/keeper/address_map_test.go

```diff
@@ -0,0 +1,113 @@
+package keeper_test
+
+import (
+	"crypto/rand"
+)
+
+const (
+	testChainID = "test-1"
+)
+
+// generateRandomBytes returns securely generated random bytes.
+// It will return an error if the system's secure random
+// number generator fails to function correctly, in which
+// case the caller should not continue.
+func generateRandomBytes(n int) ([]byte, error) {
```

Comment:
Can we make this part of utils package so it can be reused?

Reply:
https://github.com/ingenuity-build/quicksilver/pull/436/commits/e0c0614a7bc175af16ee0d302958b89021e8552a

Reply:
https://github.com/ingenuity-build/quicksilver/pull/443